docs/source/07_extend_kedro/04_plugins.md

```diff
@@ -38,7 +38,7 @@ def to_json(metadata):
 The plugin provides the following `entry_points` config in `setup.py`:
 
 ```python
-entry_points={
+entry_points = {
```

Comment:
This one should technically be without the spaces because it's specifying args within setup(), but I'm not that fussed.

Reply:
Should we do it like L94-97 below?

```suggestion
setup(
    entry_points={ ... }
)
```

Reply:
Yeah sure if you don't mind. 😊